evalml/tests/conftest.py

```diff
@@ -693,3 +693,36 @@ def _make_data_type(data_type, data):
 @pytest.fixture
 def fraud_100():
     return load_fraud(n_rows=100)
+
+
+@pytest.fixture
+def mock_imbalanced_data_X_y():
+    """Helper function to return an imbalanced binary or multiclass dataset"""
```

Comment:
Nice, this is good.  I think this is a good example to follow for our conftest fixtures.  It will help us reuse these fixtures.